performancetest/scripts/run_corfuserver.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+if [ "$JAVA_HOME" != "" ]; then
+  JAVA="$JAVA_HOME/bin/java"
+else
+  JAVA=java #java
+fi
+
+CORFUDBBINDIR="${CORFUDBBINDIR:-/usr/bin}" #/usr/bin
+CORFUDB_PREFIX="${CORFUDBBINDIR}/.." #/usr/bin/..
+
+SOURCE="${BASH_SOURCE[0]}" #./bin/corfu_server
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink get rid of soft link
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )" #/Users/lidong/vmware_proj/CorfuDB/bin
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )" #/Users/lidong/vmware_proj/CorfuDB/scripts
+
+if ls "${DIR}"/../../target/*.jar > /dev/null 2>&1; then
+  # echo "Running from development source"
+  CLASSPATH=("${DIR}"/../../infrastructure/target/infrastructure-*-shaded.jar)
+else
+  CLASSPATH=("${CORFUDB_PREFIX}"/share/corfu/lib/*.jar)
+fi
+
+# Windows (cygwin) support
+case "`uname`" in
+    CYGWIN*) cygwin=true ;;
+    *) cygwin=false ;;
+esac
+
+if $cygwin
+then
+    CLASSPATH=`cygpath -wp "$CLASSPATH"`
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651454)